src/common/valgrind_internal.h

```diff
@@ -278,6 +283,12 @@ extern unsigned _On_valgrind;
 		VALGRIND_PMC_ADD_TO_GLOBAL_TX_IGNORE(addr, len);\
 } while (0)
 
+#define DEFAULT_ENGINE "DEFAULT_REORDER"
```

Comment:
I think this belongs in `tools/pmemreorder/include/pmemreorder.h`, because it's not specific.
```
#define PMEMREORDER_ENGINE_DEFAULT ""
#define PMEMREORDER_ENGINE_FULL ""
...
```